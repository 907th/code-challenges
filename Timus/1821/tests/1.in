6
Zaitseva 21:38.2
Hauswald 21:21.0
Boulygina 22:04.4
Henkel 22:06.1
Wilhelm 21:11.1
Jonsson 22:05.8
