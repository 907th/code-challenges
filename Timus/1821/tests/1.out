3
Hauswald
Wilhelm
Zaitseva
